src/cmd/ksh93/tests/pty.sh

```diff
@@ -453,4 +453,21 @@ w fg
 u yes-yes
 !
 
+# ======
+# Test file name completion in vi mode
```

Comment:
Please add a dummy `# err_exit #` comment like on the other tests, so that shtests counts it.